pod-security/restricted/disallow-capabilities-strict/disallow-capabilities-strict.yaml

```diff
@@ -38,7 +38,7 @@ spec:
                 all:
                 - key: ALL
                   operator: AnyNotIn
-                  value: "{{ element.securityContext.capabilities.drop || '' }}"
+                  value: "{{ element.securityContext.capabilities.drop[].to_upper(@) || '[]' }}"
```

Comment:
Check [here](https://kyverno.io/policies/pod-security/restricted/disallow-capabilities-strict/disallow-capabilities-strict/). Should be backticks wrapping the brackets.